docs/content/surfacers/overview.md

```diff
@@ -112,3 +112,77 @@ surfacer {
 }
 ```
 (Source: https://github.com/google/cloudprober/blob/master/surfacers/proto/config.proto)
+
+## Additional metrics
+#### Adding your own labels
+
+ You can add additional labels to the probe results using the probe-level field, `additional_label`. An additional label's value can be static or it can be derived from the target labels.
+ 
+For the following config:
+   * if ingress target has label "fqdn:app.example.com",
+   * and prober is running in the GCE zone "us-east1-c",
+   * and prober's GCE instance has label "env:prod".
+ 
+ Probe results will look like the following:
+```
+ total{probe="my_ingress",ptype="http",metrictype="prober",env="prod",src_zone="us-east1-c",host="app.example.com"}: 90
+ success{probe="my_ingress",ptype="http",metrictype="prober",env="prod",src_zone="us-east1-c",host="app.example.com"}: 80
+```
+
+```bash
+probe {
+  name: "my_ingress"
+  type: HTTP
+
+  targets {
+    rds_targets {
+      resource_path: "k8s://ingresses"
+      filter {
+        key: "namespace"
+        value: "default"
+      }
+    }
+  }
+  
+  # Static label
+  additional_label {
+    key: "metrictype"
+    value: "prober"
+  }
+  
+  # Label is configured at the run time, based on the prober instance label (GCE).
+  additional_label {
+    key: "env"
+    value: "{{.label_env}}"
+  }
+  
+  # Label is configured at the run time, based on the prober environment (GCE).
+  additional_label {
+    key: "src_zone"
+    value: "{{.zone}}"
+  }
+  
+  # Label is configured based on the target labels.
+  additional_label {
+    key: "host"
+    value: "@target.label.fqdn@"
+  }
+
+  http_probe {}
+}
+```
+
+
+#### Adding your own metrics (external probes)
```

Comment:
It looks like it's already covered under https://cloudprober.org/how-to/external-probe/. I am not sure if we need to repeat it here.. maybe we just need to clarify it better in external probe?

Reply:
Added 2 lines to explain the label use in external probe page and put a link to that page